packages/extension/src/companion/CompanionPopupButton.tsx

```diff
@@ -48,6 +54,18 @@ export const CompanionPopupButton = (): ReactElement => {
     setShowCompanionPermission(!showCompanionPermission);
   };
 
+  useEffect(() => {
+    if (contentScriptGranted || !isFetched) {
+      return;
+    }
+
+    trackEvent({
```

Comment:
Potentially this can be called multiple times?

Reply:
This should only be called once. It will always be false once until you accept the permission. The value will never be ralse again even if we toggle the settings for disabling the companion in customize modal.